forms-web-app/src/services/task.service.js

```diff
@@ -34,12 +33,12 @@ function statusAppealStatement(appeal) {
 
 // eslint-disable-next-line no-unused-vars
 function statusOtherDocuments(appeal) {
-  return TASK_STATUS.TODO;
+  return TASK_STATUS.CANNOT_START_YET;
```

Comment:
Do we need to change the API service because currently CANNOT START YET is not available in the enum of sections states?